packages/video_player/video_player/lib/video_player.dart

```diff
@@ -475,6 +481,19 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _applyPlayPause();
   }
 
+  /// The video enters in fullscreen (only works for Web).
+  Future<void> toggleFullScreen() async {
+    if (kIsWeb) {
+      if (!value.isFullScreen) {
+        await _applyFullScreen();
+        value = value.copyWith(isFullScreen: true);
+      } else {
+        await _applyExitFullScreen();
+        value = value.copyWith(isFullScreen: false);
```

Comment:
Can't the user exit full screen directly? It seems like we would need a state listener for this, rather than managing the state from Dart.